docs/package.json

```diff
@@ -9,9 +9,10 @@
     "postflutter:build": "yarn flutter:copy",
     "flutter:clean": "rm -rf public/flutter/authenticator",
     "flutter:copy": "cp -r ./flutter/authenticator/build/web public/flutter/authenticator",
-    "build": "yarn build:props-tables && next build",
+    "build": "yarn build:props-tables && yarn lint && next build",
     "build:props-tables": "node --require esbuild-register ./scripts/generate-props-tables-data.ts",
-    "lint": "next lint",
+    "lint": "next lint && eslint . --ext ts,tsx,md,mdx",
+    "lint:fix": "eslint . --ext ts,tsx,md,mdx --fix",
```

Comment:
What do you think about removing `lint:fix` and instead we can do `yarn lint --fix`?

Reply:
Good call. I'm thinking of changing it to a pre-commit instead in a future PR